vrf/src/discrete.rs

```diff
@@ -116,34 +102,19 @@ where
         Self { T, n, K, U }
     }
 
+    /// Number of different events.
     pub fn len(&self) -> usize {
         self.n
     }
 
```

Comment:
Clippy is warning that there is not `is_empty` method

Reply:
Removed the `len` method as well.